Description: Fix UUID link reference
 On Precise it's not properly finding the UUID library.
 .
 qpid-proton (0.7-3precise) precise; urgency=medium
 .
   *  Renamed libqpid-proton2-dev package to libqpid-proton-dev.
Author: Darryl Pierce <mcpierce@gmail.com>

---
--- qpid-proton-0.7.orig/proton-c/CMakeLists.txt
+++ qpid-proton-0.7/proton-c/CMakeLists.txt
@@ -304,7 +304,7 @@ add_library (
   ${qpid-proton-platform}
   )
 
-target_link_libraries (qpid-proton ${UUID_LIB} ${SSL_LIB} ${TIME_LIB} ${PLATFORM_LIBS} rt)
+target_link_libraries (qpid-proton ${UUID_LIB} ${SSL_LIB} ${TIME_LIB} ${PLATFORM_LIBS} rt uuid)
 
 set_target_properties (
   qpid-proton
--- qpid-proton-0.7.orig/patches/series
+++ qpid-proton-0.7/patches/series
@@ -1 +1,2 @@
 fix-gettime-reference.diff
+fix-uuid-reference.diff
